content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,439 @@
+---
+linkTitle: Custom Lighting Tutorial with Flipbook Animation
+title: Custom Lighting Tutorial with Flipbook Animation
+description: A tutorial for custom lighting using a flipbook animation with six-point lighting in the Atom renderer of the Open 3D Engine (O3DE).
+toc: true
+---
+This tutorial covers how to make a material type that uses a custom surface with custom lighting, specifically for a flipbook animation with six-point lighting. This material type is especially useful for effects such as smoke and clouds to animate movement while still capturing correct lighting. Traditional 3D lighting methods do not work on 2D textures so, to apply the correct lighting, you will need a custom material type that uses custom lighting. We follow [this technique](https://realtimevfx.com/t/smoke-lighting-and-texture-re-usability-in-skull-bones/5339) to use six tangent lightmaps (top, bottom, left, right, front, and back) to approximate how a texel should be lit from any given direction. 
```

Comment:
Great ideas here. Some sentences can be rearranged to make it easier to follow. We want to clearly distinguish what we are doing and how. What is the user's one goal in this tutorial? Based on the the title, it seems that the major learning goal is applying six point lighting to an animated 2D object. How we do this is through a custom material type and shader. And the end result what we're making is a cloud/smoke effect. Right now, there's a lot going on and it's not clear to me what the goal is.

	I also think there's a lot of terminology here, so I'm not sure what to call the material type we are creating. (E.g. Six-point lighting, flipbook animation, custom lighting, custom surface.) Later, you call this six-point lighting material type, which clears it up. But it would help if that was clear from the first paragraph.

Reply:
This makes sense, I tried to consolidate the ideas into one sentence and then make another paragraph for the rest of the information:

> This tutorial covers how to make a six-point lighting material type by writing custom shaders to apply lighting to an animated 2D object, creating a cloud/smoke effect.

> Traditional 3D lighting methods do not work on 2D textures so, to apply the correct lighting, you will need a material type that uses a custom surface with custom lighting. We follow [this technique](https://realtimevfx.com/t/smoke-lighting-and-texture-re-usability-in-skull-bones/5339) to use six tangent lightmaps (top, bottom, left, right, front, and back) to approximate how a texel should be lit from any given direction.

Reply:
This is great and reads a lot better :) 

You can remove the link to the technique. This technique's been used and presented in the VFX industry before, so the post isn't the main source of the material.